backend/Services/Services/RestaurantService.cs

```diff
@@ -56,12 +57,12 @@ public void DeleteAccount(Credentials creds)
 
             if (restaurant == null)
             {
-                throw new Exception("The user does not exist");
+                throw new EntityNotFoundException("The user does not exist");
             }
 
             if (!restaurant.Credentials.Equals(creds))
```

Comment:
won't this always be false? because unhashed credentials password will never equal hashed restaurant credentials password